plugin/java-testing/src/test/java/com/buschmais/jqassistant/plugin/junit/test/rule/RuleSetIT.java

```diff
@@ -0,0 +1,72 @@
+package com.buschmais.jqassistant.plugin.junit.test.rule;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.buschmais.jqassistant.core.report.api.model.Column;
+import com.buschmais.jqassistant.core.report.api.model.Result;
+import com.buschmais.jqassistant.core.report.api.model.Row;
+import com.buschmais.jqassistant.core.rule.api.model.Concept;
+import com.buschmais.jqassistant.plugin.java.api.model.MethodDescriptor;
+import com.buschmais.jqassistant.plugin.java.api.model.TypeDescriptor;
+import com.buschmais.jqassistant.plugin.java.test.AbstractJavaPluginIT;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import static com.buschmais.jqassistant.core.report.api.model.Result.Status.SUCCESS;
+import static com.buschmais.jqassistant.plugin.java.test.assertj.MethodDescriptorCondition.methodDescriptor;
+import static com.buschmais.jqassistant.plugin.java.test.assertj.TypeDescriptorCondition.typeDescriptor;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RuleSetIT extends AbstractJavaPluginIT {
+
+    @Test
+    void providedConceptAssertMethod() throws Exception {
```

Comment:
This test method should be part of AssertJIT and MockitoIT (to ease later separation of frameworks if necessary)

Reply:
Done.